.github/actions/dockerfiles/Dockerfile.alpine-binary

```diff
@@ -23,5 +23,18 @@ RUN case ${TARGETPLATFORM} in \
     && unzip ${BIN_ARCH}.zip -d /out
 
 FROM --platform=${TARGETPLATFORM} alpine
-COPY --from=builder /out/stacks-node /out/stacks-signer /bin/
-CMD ["stacks-node", "mainnet"]
+COPY --from=builder /out/* /bin/
+ARG TAG
+
+RUN case "${TAG}" in \
+        signer-*) \
+            echo "/bin/stacks-signer run --config /signer-config.toml" > /tmp/command.sh \
+            ;; \
+        *) \
+            echo "/bin/stacks-node mainnet" > /tmp/command.sh && \
+            rm /bin/blockstack-cli /bin/clarity-cli /bin/relay-server /bin/stacks-events /bin/stacks-inspect \
```

Comment:
Don't we want to remove the `stacks-signer` binary here, too?

Reply:
i don't think there's a need to - since the signer binary won't exist, that conditional won't be met. 
technically it's not needed, and i can remove/revert but i don't *think* it will cause any issues

Reply:
actually i misunderstood what you meant - in this case though, no the signer won't be in the archive we're using in that dockerfile. see the files we use to build the binary archive, ex: https://github.com/stacks-network/actions/blob/feat/specific_binaries_stacks-core/stacks-core/create-source-binary/build-scripts/Dockerfile.linux-glibc-arm64#L24

```
    && case "${TAG}" in \
        signer-*) \
            BUILD_ARG="--bin stacks-signer" \
            ;; \
        *) \
            BUILD_ARG="--bin stacks-node --bin stacks-inspect --bin clarity-cli --bin blockstack-cli" \
            ;; \
    esac \
 ```
since we're not building a `signer` release, the `*` section of that case should be the one that is used, which has a hardcoded set of binaries to build.

Reply:
*if* we were using the archive build files in `main` - you would be correct though: https://github.com/stacks-network/actions/blob/main/stacks-core/create-source-binary/build-scripts/Dockerfile.linux-glibc-arm64#L24

Reply:
Ah I gotcha - makes sense!